src/Groups/perm.jl

```diff
@@ -156,7 +156,7 @@ perm(g::PermGroup, L::AbstractVector{<:fmpz}) = perm(g, [Int(y) for y in L])
 
 function (g::PermGroup)(L::AbstractVector{<:IntegerUnion})
    x = GAP.Globals.PermList(GAP.GapObj(L;recursive=true))
-   if length(L) <= degree(g) && GAPWrap.IN(x,g.X)
+   if GAP.Globals.LargestMovedPoint(x) <= degree(g) && GAPWrap.IN(x, g.X)
```

Comment:
But this way, we now would accept `G([2,3,1,4,6,5,7])` when `G` is `sym(6)` (this is from our test suite). But arguably it shouldn't, the list was too long?

In any case, I am not sure which aspect of issue #894 this is supposed to address? Perhaps that `ans([1,2,3])` doesn't work in that example? But there the bug is that `ans` is a subgroup that has the wrong degree, isn't it?